src/SFML/Graphics/Shape.cpp

```diff
@@ -143,6 +144,53 @@ FloatRect Shape::getGlobalBounds() const
     return getTransform().transformRect(getLocalBounds());
 }
 
+Vector2f Shape::getGeometricCenter() const
```

Comment:
This implementation could be moved before `getLocalBounds` to better match the order of methods in the class definition.